Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.cpp

```diff
@@ -788,7 +784,7 @@ void SkylanderPortalWindow::LoadSkylander(u8 slot)
 
 void SkylanderPortalWindow::LoadSkylanderPath(u8 slot, const QString& path)
 {
-  File::IOFile sky_file(path.toStdString(), "r+");
+  File::IOFile sky_file(path.toStdString(), "rb");
```

Comment:
This breaks saving. Fix here: https://github.com/deReeperJosh/dolphin/pull/4

Reply:
Have made this fix within my latest changes :)